tests/ProjectCheckerTest.php

```diff
@@ -124,8 +124,7 @@ public function testAfterCodebasePopulatedIsInvoked(): void
             /** @var bool */
             public static $called = false;
 
-            /** @return void */
-            public static function afterCodebasePopulated(AfterCodebasePopulatedEvent $event)
+            public static function afterCodebasePopulated(AfterCodebasePopulatedEvent $event): void
```

Comment:
I suspect it'll fail on PHP <7.4

Reply:
Good catch! I thought I had all of these. I guess this one slipped through.
Arghl, found another one…

Reply:
Always good to have a good an excellent reviewer :relieved:
Those interfaces shouldn’t differ in using or not using a return type.